### PR TITLE
Added documentation for Wiggle Sort

### LINKED ISSUE #374: Article for Wiggle sort
## 📚 Documentation

Article for Wiggle sort

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

Please assign this to me under GSSOC'21.

### PR DESCRIPTION
**Pull request by nidh-ish**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added documentation for Wiggle Sort.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #374

### CODE REVIEW

File: docs/sorting-algorithms/wiggle-sort.mdx [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+title: Wiggle Sort
+slug: /sorting-algorithms/wiggle-sort
+---
+
+## Introduction
+
+Given an unsorted array `A[n]`, Wiggle Sort algorithm sorts the array such that `A[0] < A[1] > A[2] < A[3] > A[4] . . .`.
```
Comment (HarshCasper) [inline docs/sorting-algorithms/wiggle-sort.mdx:8]: Give a better introduction. It lacks any intuition as of now.

File: docs/sorting-algorithms/wiggle-sort.mdx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+## Explanation
+
+
```
Comment (HarshCasper) [inline docs/sorting-algorithms/wiggle-sort.mdx:12]: Remove extra lines here

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Also make sure you remove redundant comments here and there.

**Comment by nidh-ish** [comment]
@HarshCasper I have updated the documentation. Please have a look.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**